stats_report.h

```diff
@@ -0,0 +1,74 @@
+class SystemReport {
```

Comment:
Is this not missing a bunch of defines? E.g. stdint? I know it works because you include it after mbed.h in main.cpp, but this is not copy-paste friendly.

Reply:
Fair enough, I can add update that.